### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by Harsh4teams**

### CODE REVIEW

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -511,14 +510,12 @@ The following are known issues and limitations for calling and meetings:
 
 - Interoperability with Skype for Business is limited to audio calls; there is no video modality.
 - Incoming and outgoing video stream resolution is limited to 720p resolution.
-- Only one video stream from an incoming camera or screen share stream is supported. When there's an incoming screen share, that screen share is shown, instead of the video of the dominant speaker.
 - Teams doesn't switch to use the last audio device that a user selected, if the device is disconnected, and then reconnected.
 - Live events are not optimized.
 - Outgoing screen sharing:
-  - Application sharing is not supported.
+  - Application sharing is not supported for VMware and AVD/W365
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:516]:
```suggestion
  - Application sharing is not supported for VMware and AVD/W365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Harsh4teams, kindly commit the minor suggested change. Many thanks.

**Comment by Harsh4teams** [comment]
Thanks. I have made the suggested changes.

**Comment by Uralwiki** [comment]
Open

**Pull request merged**